pkg/agent/datamodel/types.go

```diff
@@ -1503,6 +1503,7 @@ type NodeBootstrappingConfiguration struct {
 	IsARM64                        bool
 	CustomCATrustConfig            *CustomCATrustConfig
 	DisableUnattendedUpgrades      bool
+	DisableSSH                     bool
```

Comment:
can we make this an enum or string type with specific values, to differentiate `false` vs `unspecified` (e.g. enabled, disabled, and default is empty string/unspecified)?

in the near future when we try to support in-place update, we will care about this